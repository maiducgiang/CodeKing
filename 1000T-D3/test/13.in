56
75094 98128 61169 42942
24464 79414 64379 7430
38436 1194 21364 71350
17793 11883 91220 87020
82260 88486 70917 10103
89349 15374 67245 13386
99603 3330 60852 28147
56308 42843 13063 91468
77371 7740 21262 1198
34855 83596 48157 46788
23801 70604 36474 13953
13822 37274 42771 81387
25948 40994 74787 28413
92061 71897 75324 97676
35100 65708 78485 25810
41196 98283 23504 34595
6631 61834 99787 12319
24779 60767 69017 36143
46440 85555 17970 91623
89075 54197 36418 78000
47614 36249 47064 22890
7053 48192 9160 4694
54696 25747 43500 69917
89392 5218 57872 51872
14954 12465 72533 30867
83135 1585 1364 33054
16761 16520 66071 98519
73894 32930 25700 26244
13766 36816 86207 65555
44076 28075 62177 4052
61043 81865 64713 86326
91924 86833 6139 35967
17255 39810 13800 81233
47857 2320 88056 4061
84863 5862 81614 88350
61765 18009 45017 81226
99155 97116 74425 73590
11066 35636 87326 65615
45860 64355 88022 25599
69838 26127 81550 36271
99606 70527 83121 60395
37515 11709 84688 52897
79735 10263 82566 78389
76309 9098 3008 30832
20886 61035 29491 64273
60713 77479 36013 65446
70354 46657 71030 28565
87502 2045 78660 65248
54334 60808 34482 31529
26877 70381 21223 37048
72853 80933 91376 66927
43028 83566 77386 31616
88282 7586 75675 43621
22289 25385 51579 35080
95488 52865 41259 24847
49714 36391 23703 81200
