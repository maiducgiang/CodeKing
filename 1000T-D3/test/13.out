4633653294
3719393570
846655320
1573168860
1272602022
5180245239
5952134106
1295209513
1489766842
3978016077
2039187975
2127996606
1862505726
8541444508
3308199880
2766948468
5590229050
3379371193
2371950150
5217679696
1364621970
409940022
3059434499
5141985824
565291792
115558080
1643456991
1916989720
2889501480
1108654077
6617052104
1097389223
787497000
4019227592
6965495314
3432562886
7298519015
2871695410
2801937030
4512284467
6676553724
2804829501
6540531459
256904256
2415932211
4577216956
3015955115
6855479780
1936333154
1919024488
5614150811
4497446916
6437578706
941586704
3072119012
2040946815
